orchagent/pfcwdorch.cpp

```diff
@@ -97,6 +97,7 @@ PfcWdAction PfcWdOrch<DropHandler, ForwardHandler>::deserializeAction(const stri
     {
         { "forward", PfcWdAction::PFC_WD_ACTION_FORWARD },
         { "drop", PfcWdAction::PFC_WD_ACTION_DROP },
+        { "alert", PfcWdAction::PFC_WD_ACTION_ALERT },
```

Comment:
is it better to move this const into the header file?

Reply:
It is not intended to be used by anyone else. Enum is in header.

Reply:
then a const static is better? do you know if the compiler will optimize here to only initialize the variable once?

Reply:
const static will be better here. Move out of this function, and make it a cpp global is even better.

Reply:
@qiluo-msft thanks for the explanation.